pkg/version/version.go

```diff
@@ -40,6 +40,8 @@ var (
 	buildMetaFile = "/src/github.com/openebs/maya/BUILDMETA"
 )
 
+// GetVersion returns the version from the global Version variable,
+// or if this unset, from the VERSION file at the root of the repo
```

Comment:
Rewrite `if Version is unset then from the VERSION file at the root of the repo`

Reply:
pushed 2nd commit with rewording - thanks.